frontend/app/src/locales/en.json

```diff
@@ -3202,7 +3202,7 @@
     "manage_assets": "Manage Assets",
     "manage_assets_sub": {
       "custom_assets": "Custom Assets",
-      "managed_assets": "Managed Assets",
+      "managed_assets": "Assets",
```

Comment:
Shouldn't you also change the json key? Otherwise developers will wonder why the name difference in the key and the value in the future.

Reply:
on it now.